### ISSUE TITLE
Implement email notifications for admins

### ISSUE THREAD

**Issue by PatrikD** [sudoroom/sudo-humans#53]
Admins should be notified whenever a new member joins, starts or stops a payment plan, or when a credit card is bouncing (perhaps only after the 3rd failure?)

Not all admins may want to receive all these notifications, so perhaps the profile page for admins could include a series of checkboxes to sign up to these various alerts.

**Comment by PatrikD** [comment]
Do the individual members even get notified when their credit card starts bouncing? That obviously would  be a must-have feature as well!

**Comment by rcsheets** [comment]
No, I don't think anyone gets notified ever. The two features are related but should probably be tracked separately.

**Comment by okdistribute** [comment]
I started a branch here that is the start to this implementation for emailing when users subscribe. https://github.com/sudoroom/sudo-humans/tree/payment-email

To find out when a payment bounces, we need to set up a Stripe webhook. Who has access to the stripe account? https://stripe.com/docs/recipes/sending-emails-for-failed-payments

**Comment by rcsheets** [comment]
I had assumed we could poll for that, but a webhook sounds like a better option.
I don't have Stripe account access, but I should, so I'll follow up on that.

**Comment by rcsheets** [comment]
I have access to Stripe now.
